episode-6.msrt

```diff
@@ -393,8 +387,7 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [Version 1:] [sarcastically] Ah, rejoice Carrot. It seems nobody will see your wondrous potion...
-[en] [Version 2:] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
```

Comment:
OK (no significant arguments against version 1 - just that "rejoice Carrot" doesn't feel like something Pepper would say I may just change that part)